APCAppCore/APCAppCore/UI/TabBarControllers/Profile/APCProfileViewController.m

```diff
@@ -1183,23 +1165,18 @@ - (void) imagePickerControllerDidCancel:(UIImagePickerController *)picker
 
 - (void)taskViewController:(ORKTaskViewController *) __unused taskViewController didFinishWithReason:(ORKTaskViewControllerFinishReason)reason error:(nullable NSError *)error
 {
-    if (reason == ORKTaskViewControllerFinishReasonCompleted)
-    {
+    if (reason == ORKTaskViewControllerFinishReasonCompleted) {
```

Comment:
Hm. This is a peculiar spacing rule. I'm going to accept the patch anyway, but future patches should adopt one of the two possible patterns below:

In ResearchKit we do this:

```
if (a) {
    b;
} else {
    c;
}
```

In this codebase it's a mixture of that and 

```
if (a)
{
    b;
}
else
{
    c;
}
```